spec/support/helpers/github_api_helper.rb

```diff
@@ -15,6 +15,28 @@ def stub_pull_request_files_request(repo, pull_request)
     )
   end
 
+  def stub_non_existant_pull_request_request(repo, pull_request)
+    stub_request(
+      :get,
+      url(repo, "/pulls/#{pull_request}")
+    ).with(headers: request_headers).to_return(
+      status: 404,
+      body: '',
+      headers: {}
+    )
+  end
+
+  def stub_pull_request_request_with_fixture(repo, pull_request, fixture:)
+    stub_request(
+      :get,
+      url(repo, "/pulls/#{pull_request}")
+    ).with(headers: request_headers).to_return(
+      status: 200,
+      body: File.read("spec/support/fixtures/#{fixture}.json"),
```

Comment:
There's only one pull request fixture available, no need to get its name from an argument.